README.md

```diff
@@ -41,6 +41,55 @@ source zephyr/zephyr-env.sh
 cd $MODULE.git
 ```
 
+### Build & Flash
+
+**Build and flash targets with `west`**
+
+> [!NOTE]
+> Below, we build with [application overlay](https://docs.zephyrproject.org/latest/develop/application/index.html#important-build-system-variables)
+> files to support `tt-console` from firmware.
+
+```shell
+# Build `tt-console`
+gcc -Iinclude -std=gnu11 -o tt-console scripts/tt-console/console.c
+
+# Build and flash SMC firmware
+west build --sysbuild -p -b tt_blackhole@p100/tt_blackhole/smc app/smc -- -DEXTRA_DTC_OVERLAY_FILE="vuart.overlay" -DEXTRA_CONF_FILE="vuart.conf"
+./scripts/bmc-reset.py
+west flash
```

Comment:
We might want to specify to use the `tt_flash` west runner here. What do you think @danieldegrasse ?

Reply:
Yes, I would argue for `west flash -r tt_flash --force` here